SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.cpp

```diff
@@ -34,6 +35,12 @@ int32 UGenerateSchemaAndSnapshotsCommandlet::Main(const FString& Args)
 
 	FSpatialGDKEditor SpatialGDKEditor;
 
+	if (!HandleDeleteSchemaOption(Switches))
```

Comment:
Having HandleDeleteSchemaOption return true even if it didn't delete the database (because the option wasn't specified) seems a little strange. How about each commandlet has a HandleOptions function, and in it you check for the flag before attempting to delete the database. It could take/return an abort flag.